pinot-tools/src/main/java/org/apache/pinot/tools/QuickStartBase.java

```diff
@@ -83,6 +83,7 @@ public abstract class QuickStartBase {
 
   protected static final Map<String, String> DEFAULT_STREAM_TABLE_DIRECTORIES = ImmutableMap.<String, String>builder()
       .put("airlineStats", "examples/stream/airlineStats")
+      .put("dailySales", "examples/stream/dailySales")
```

Comment:
nit: In the future we will need to split these large quickstarts, given they are growing too much and take a bit to start.